core/src/main/java/com/training/core/servlets/GetAllRegistrationDataServlet.java

```diff
@@ -126,9 +94,14 @@ private void setMessageAndSendResponse(String message, List<Node> userRegistrati
 
             for (Node node:userRegistrationDataList){
 
-                Property references = node.getProperty("hobbies");
-                Value[] values = references.getValues();
-                String hobbies = values[0].getString();
+                if (node.hasProperty("hobbies")){
+                    Property references = node.getProperty("hobbies");
+                    Value[] values = references.getValues();
+                    hobbies = values[0].getString();
+                }else {
+                    hobbies="--";
```

Comment:
don't add this double lines just show empty if it's not there

Reply:
fixed